### ISSUE TITLE
Comment for HW1

### ISSUE THREAD

**Issue by rickjunli** [jduyan/DAT_SF_16#2]
I tried Lumosity before and I liked it! Nice product.

**Comment by ghego** [comment]
Yes, I have tried it. Pretty cool! (not sure why you're mentioning it here)

BTW congrats! You found how to cite us!
Keep rocking!!

(you can close the issue now)

**Comment by rickjunli** [comment]
@ghego: Ah sorry! I should have realized that the reason wouldn't be apparent to anyone else besides Josh. Josh included a few links about himself in his readme file including Linkedin. From there I found out that he works for Lumosity. That's why I mentioned here. :)

**Comment by ghego** [comment]
I was just one link away from it :)
